learners/setup.md

```diff
@@ -16,30 +16,21 @@ e.g. your Desktop or a folder you have created for using in this workshop.
 
 ## Software
 
-1. Download and install the latest [Anaconda
-   distribution](https://www.anaconda.com/download/) for your
-   operating system. Make sure to choose the Python 3 version (as
-   opposed to the one with Python 2). If you wish to use an existing
-   installation, be sure to upgrade your scikit-image to at least 0.19.
-   You can upgrade to the latest scikit-image using the shell command that follows.
+1. Download and install the latest [MiniForge distribution of Python](https://conda-forge.org/download/) for your operating system. 
```

Comment:
I'm not sure it's obvious that, on Unix-like platforms, you need to run
```bash
bash Miniforge3-$(uname)-$(uname -m).sh
```
from the terminal to *install* Miniforge... I would add a line about this. Is it obvious on Windows? I guess so, if you need to double-click what you just downloaded.

Reply:
Yes - on Windows it's an `.exe` file, so you just double-click the downloaded file.

Reply:
I added a link to the central instructions, which are more detailed, in https://github.com/datacarpentry/image-processing/pull/342/commits/9c8345e10faaa2ff86260d63e86f37539cd0642d.